Apps/WebClient/src/appsettings.json

```diff
@@ -41,33 +41,34 @@
             "disabled": false
         }
     ],
-    "WebClient": {
-        "RedirectToWWW": true,
-        "RegistrationStatus": "open",
-        "LogLevel": "Debug",
-        "TimeOuts": {
-            "Idle": "900000",
-            "LogoutRedirect": "20000",
-            "ResendSMS": "1"
-        },
-        "ExternalURLs": {
-            "HealthLinkImmunizationSchedule": "https://www.healthlinkbc.ca/tools-videos/bc-immunization-schedules",
-            "HealthLinkVaccineSearch": "https://www.healthlinkbc.ca/search/"
-        },
-        "Modules": {
-            "Immunization": true,
-            "Medication": true,
-            "Laboratory": true,
-            "Encounter": false,
-            "Note": true,
-            "Comment": true,
-            "CovidLabResults": true,
-            "Dependent": true
-        },
-        "HoursForDeletion": 720,
-        "MinPatientAge": 12,
-        "MaxDependentAge": 12
+  "WebClient": {
+    "RedirectToWWW": true,
+    "RegistrationStatus": "open",
+    "LogLevel": "Debug",
+    "TimeOuts": {
+      "Idle": "900000",
+      "LogoutRedirect": "20000",
+      "ResendSMS": "1"
+    },
+    "ExternalURLs": {
+      "HealthLinkImmunizationSchedule": "https://www.healthlinkbc.ca/tools-videos/bc-immunization-schedules",
+      "HealthLinkVaccineSearch": "https://www.healthlinkbc.ca/search/"
+    },
+    "Modules": {
+      "Immunization": true,
+      "Medication": true,
+      "Laboratory": true,
+      "Encounter": false,
+      "Note": true,
+      "Comment": true,
+      "CovidLabResults": true,
+      "Dependent": true
     },
+    "HoursForDeletion": 720,
+    "MinPatientAge": 12,
+    "MaxDependentAge": 12,
+    "EmailVerificationExpiryHours": 5
```

Comment:
EmailVerificationExpiryHours => MessageVerificationExpire
Also, it should be in seconds

Reply:
I don't agree with the naming here as it is not Message but Email, the name should be meaningful. MessageVerificationExpiry doesn't tell much about if it is in minutes, hours, days, months.
Also, why it should be in seconds? we need to display this as is in the email template, why do we need to convert seconds to hours? Unless the requirement asks to allow email to expire in seconds or minutes which is not the case here right?

Reply:
Seconds is part of the requirements. The name is arguable

Reply:
ah you re right for the seconds.
We need team discussion for the naming :-)

Reply:
You can propose a different name and we can agree to that one, too